components/our_club_login.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "our_club_login",
+  "owner": ["@jcgoette"],
+  "manifest": "https://raw.githubusercontent.com/jcgoette/our_club_login_homeassistant/main/custom_components/our_club_login/manifest.json",
```

Comment:
`requests` is a core requirement of Home Assistant, you should not have this in your manifest file, since that is the only requirement for this integration once that is removed this file is no longer needed.